provisionerd/proto/version.go

```diff
@@ -15,16 +15,18 @@ import "github.com/coder/coder/v2/apiversion"
 //
 // API v1.5:
 //   - Add new field named `prebuilt_workspace_build_stage` enum in the Metadata message.
+//   - Add new field named `running_agent_auth_tokens` to provisioner job metadata
+//   - Add new field named `resource_replacements` in PlanComplete & CompletedJob.WorkspaceBuild.
+//   - Add new field named `api_key_scope` to WorkspaceAgent to support running without user data access.
+//
+// API v1.6:
```

Comment:
`plan` shipped in v1.5, but `module_files` did not.

Reply:
Oh, that change must've snuck in last night; I thought the release was cut already. Thanks 👍